cdf_dump/README.md

```diff
@@ -0,0 +1,186 @@
+# cdf_dump CLI tool
+
+The cdf_dump utility is a simple CLI wrapper for [Cognite Spark Data Source](https://github.com/cognitedata/cdp-spark-datasource)
+to simplify copying of large amounts data from CDF.
+
+## Installation
+
+Download the cdf_dump.jar from https://github.com/cognitedata/cdp-spark-datasource/releases and run it from command line: `java -jar cdf_dump.jar`. The cdf_dump.jar file contains all dependencies, including Apache Spark, so it's bit large, but should not require anything except the JVM on your OS.
+Spark currently only supports Java 11 and Java 8, so cdf_dump also won't work on other JVM versions.
+If you have the choice, we recommend Java 11. You may use GraalVM, but it doesn't seem to improve performance much.
+
+
+## Examples
+
+```shell
+cdf_dump --raw my-db.table -o out_directory
+```
+
+Copies my-db.table into `out_directory/raw/my-db/table` directory in JSON format. You can use [`jq`](https://stedolan.github.io/jq/) to process the data further.
+
+```shell
+cdf_dump --assets --events
```

Comment:
forgot `-o out-directory` ? also does it create the directory for us if it does not exist? and  mentioning file names might be cool.

Reply:
hmm file names are not necessary since all types have their own folder but the rest of my question remains.

Reply:
`-o out` is default. And yes, it creates all these directories automatically.

Output files would be nice, but Spark does not want to write a specific file, it intends on creating stuff like `raw/test/cpu-count2/part-00000-56d7947b-b344-4341-abb9-8866431afcf0-c000.json`...